plugins/go-build/test/goenv-install.bats

```diff
@@ -162,8 +162,10 @@ OUT
 1.15.6
```

Comment:
You need an additional 2 sets to pass the CI like this.
https://github.com/syndbg/goenv/pull/160/files

Reply:
Added, thanks